docs/concepts/composition.md

```diff
@@ -161,813 +101,157 @@ spec:
-    finalizer.apiextensions.crossplane.io
-  Generation:        3
-  Resource Version:  1428420
-  UID:               d87e9580-9d2e-41a7-a198-a39851815840
-Spec:
-  Composition Selector:
-    Match Labels:
-      Provider:  azure
-      Purpose:   example
-  Parameters:
-    Location:    au-east
-    Storage GB:  20
-    Version:     5.7
-  Resource Ref:
-    API Version:  example.org/v1alpha1
-    Kind:         CompositeMySQLInstance
-    Name:         default-example-8t4tb
-  Write Connection Secret To Ref:
-    Name:  example-mysqlinstance
-Status:
-  Address:    example.mysql.database.azure.com
-  Admin DSN:  mysql://admin@example.mysql.database.azure.com:3306/my-database-name
-  Conditions:
-    Last Transition Time:  2020-05-15T07:26:49Z
-    Reason:                Resource is available for use
-    Status:                True
-    Type:                  Ready
-    Last Transition Time:  2020-05-15T07:08:11Z
-    Reason:                Successfully reconciled resource
-    Status:                True
-    Type:                  Synced
-  Connection Details:
-    Last Published Time:  2020-05-15T07:08:11Z
-Events:
-  Type    Reason                      Age                    From                                       Message
-  ----    ------                      ----                   ----                                       -------
-  Normal  ConfigureCompositeResource  8m23s                  claim/compositemysqlinstances.example.org  Successfully configured composite resource
-  Normal  BindCompositeResource       8m23s (x7 over 8m23s)  claim/compositemysqlinstances.example.org  Composite resource is not yet ready
-  Normal  BindCompositeResource       4m53s (x4 over 23m)    claim/compositemysqlinstances.example.org  Successfully bound composite resource
-  Normal  PropagateConnectionSecret   4m53s (x4 over 23m)    claim/compositemysqlinstances.example.org  Successfully propagated connection details from composite resource
+    name: my-db-connection-details
 ```
 
-## Current Limitations
-
-At present the below functionality is planned but not yet implemented:
-
-* Compositions are mutable, and updating a composition causes all composite
-  resources that use that composition to be updated accordingly. Revision
-  support is planned per issue [#1481].
-
-Refer to the list of [composition related issues] for an up-to-date list of
-known issues and proposed improvements.
-
-[Current Limitations]: #current-limitations
-[Infrastructure Composition Concepts]: composition-concepts.png
-[structural schemas]: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#specifying-a-structural-schema
-[Infrastructure Composition Provisioning]: composition-provisioning.png
-[composition related issues]: https://github.com/crossplane/crossplane/labels/composition
-[#1481]: https://github.com/crossplane/crossplane/issues/1481
+> We designed Crossplane to be a tool platform teams can use to offer
```

Comment:
No real suggestion here just a comment that I had to read it twice to understand this section.

Reply:
I was on the fence about including this at all - I had copied it from the terminology guide when I was trying to frame up the claims section. I've removed part of it that I didn't think was very useful, and merged the "public and private" analogy part into the opening paragraph.